sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/sink/CosmosSinkTask.java

```diff
@@ -77,6 +96,12 @@ record -> this.sinkTaskConfig
                     .getDatabase(this.sinkTaskConfig.getContainersConfig().getDatabaseName())
                     .getContainer(containerName);
 
+            CosmosThroughputControlHelper
+                .tryEnableThroughputControl(
```

Comment:
Can this throw an error which we are not handling here?

Reply:
very rarely, but possible. I think it will be decided whether we want to suppress any throughput control exception. In azure-cosmos, we allow customer to config continueOnInitError in throughput control config to control the behavior.  For Kafka, I think we can start with the default (fail at error), and then if needed, in future add another config to support this extra behavior.

Reply:
And also for the above question:
```
What is the downside of having this config enabled for all containers?
Ideally we should, however, we can do that post GA, thoughts?
```

you mean enable by default? but we can not do this it has to be an opt-in behavior as we need customer to provide the throughput control container